### PR TITLE
Keep request as part of pyramid render kwargs

### LINKED ISSUE #385: Incorrect render patching in pyramid.
https://github.com/DataDog/dd-trace-py/blob/261136e112b23862a78308a2423e15364ae4aaa6/ddtrace/contrib/pyramid/trace.py#L31

Here we're removing request from kwargs but pyramid's render has a request kwarg so we need to keep it.

### PR DESCRIPTION
**Pull request by joual**
Fixes #385 

cc @sudarkoff

### CODE REVIEW

File: ddtrace/contrib/pyramid/trace.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def includeme(config):
 
 def trace_render(func, instance, args, kwargs):
     # If the request is not traced, we do not trace
-    request = kwargs.pop('request', {})
+    request = kwargs.get('request', {})
```
Comment (palazzem) [inline ddtrace/contrib/pyramid/trace.py]: definitely, it must be a `get`

File: tests/contrib/pyramid/test_pyramid.py [hunk 2]
```diff
@@ -118,6 +119,29 @@ def test_json(self):
         eq_(s.error, 0)
         eq_(s.span_type, 'template')
 
+    def test_renderer(self):
```
Comment (palazzem) [inline tests/contrib/pyramid/test_pyramid.py]: thanks for adding a regression test!

**Comment by joual** [comment]
@palazzem I just rebased against master :)

**Comment by palazzem** [comment]
Thank you @joual ! let's wait for the CI so I can merge the PR!

**Comment by palazzem** [comment]
@joual thank you very much for your contribution!

**Pull request merged**